common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -434,33 +473,36 @@
             1
         ],
         "import/no-named-as-default-member": [
-            1
+            "off"
+        ],
+        "import/no-nodejs-modules": [
+            "warn"
         ],
         "import/no-unassigned-import": [
             "error"
         ],
         "import/no-unresolved": [
-            2,
+            "error",
             {
                 "caseSensitive": true,
                 "caseSensitiveStrict": false
```

Comment:
Seems reasonable to error for no-unresolved, maybe also enable caseSensitiveStrict?